docs/documentation/server_admin/topics/account.adoc

```diff
@@ -158,4 +134,4 @@ If you select *Direct membership* checkbox, you will see only the groups you are
 * You need to have the *view-groups* account role for being able to view *Groups* menu.
 
 .View group memberships
-image:images/groups_account_console.png[View group memberships]
+image:images/account-console-groups.png[View group memberships]
```

Comment:
```suggestion
.View group memberships
image:images/account-console-groups.png[View group memberships]
```

Reply:
@andymunro not sure what has changed here, that same header already exists.